apps/website/src/pages/api/notifications/batched-create-notification-pushes.ts

```diff
@@ -31,7 +31,7 @@ export default createTokenProtectedApiHandler(
       }
 
       const calls: Array<Promise<Response>> = [];
-      options.subscriptionIds.forEach((subscriptionId) => {
+      for (const subscriptionId of options.subscriptionIds) {
```

Comment:
Could this not just be a map?

Reply:
Don't know if I understand what you want to do, but for…of is used because of readability and performance. https://biomejs.dev/linter/rules/no-for-each/

Reply:
We have a for loop iterating over a thing, and pushing one item to an array for each interaction. Instead of creating an empty array and then pushing to it, just create the array using map, and then we don't need the for loop?

Reply:
like [this](https://github.com/alveusgg/alveusgg/pull/644/commits/8c6a9266293e8c40ebaad222ad3f70f5dd022985)?

Reply:
`(subscriptionId) => {\ncallEndpoint(` -> `(subscriptionId) => callEndpoint(`. Before the code was pushing the result of `callEndpoint`, now you're pushing `void` (you're calling the method and not returning anything)